api.bs

```diff
@@ -463,7 +467,7 @@ navigator.privateAttribution.saveImpression({
 <xmp class=idl>
 dictionary PrivateAttributionImpressionOptions {
   required unsigned long histogramIndex;
-  required unsigned long filterData;
+  unsigned long filterData;
```

Comment:
Do you want to make this a separate change?  Making this optional is probably fine, I'd like that to be an explicit choice.

Reply:
I intended this to be a typographical correction because it's already described as optional in the text that follows. But even if so, I agree it's better to consider separately from this PR.

Reply:
https://github.com/patcg/ppa-api/pull/57 for making it optional here. I also filed an issue related to the type, https://github.com/patcg/ppa-api/issues/58.